docs/intro/install.md

```diff
@@ -13,6 +13,12 @@ Starport is written in the Go programming language. To use Starport on a local i
 
 - [Golang >=1.16](https://golang.org/)
 
+## Upgrading
+
+Remove any previous Starport installation. To do so, run `rm $(which starport)` command with or without `sudo` depending on your user and repeat until there is no `starport` binary left in your system.
+
+Then follow the regular installation instructions.
```

Comment:
```suggestion
## Upgrading Your Starport Installation

Before you install a new version of Starport, remove all existing Starport installations. 

To remove the current Starport installation:

1. Run the `which starport` command to locate the Starport executable file.  By default, the Starport binary is installed in `/usr/local/bin`.

1.  Change to the directory where Starport is installed. For example:

    ```bash
    cd /usr/local/bin
    ```


1.  Run the remove command.  Depending on your user permissions, run the command with or without `sudo`:

    ```bash
    rm starport
    ``` 
    
1. Repeat these steps until all `starport` installations are removed from your system.

After all existing Starport installations are removed, follow the Starport installation instructions.

Reply:
@ilgooz I think ^^ unexpected installation happened before I knew better than to install go using brew. I had issues with the PATH variables. 
Since then, I developed: https://github.com/cosmos/sdk-tutorials/blob/master/TECHNICAL-SETUP.md for onboarding awesomeness.

Reply:
Yep, that usually happens. Different kind of installation options (curl, homebrew, git clone & make install) adds the Starport binary to a different `bin` folder and users usually forget about them.

Even if you remove a single installation and install a new one, you may end up using another old version of Starport that is still not removed from your system. It is all about the order of presence of the `bin` dirs in $PATH. This is why I think we should recommend users to uninstall any previous installations.

Reply:
> I was surprised to find an unexpected Starport installation:

`starport` is installed in `~/go/bin` when you build it from source, so that's expected.

Reply:
Two things I would fix from your change requests:

*  `/usr/local/bin` is not the default bin dir for everyone. It depends on the system. Maybe we should rm this.
* We don't need two step process for removing the binary by first cd'ing to dir. We can just recommend `rm $which starport)`.